diff -aurN bonfire-0.3.0.orig/src/burn-dialog.c bonfire-0.3.0/src/burn-dialog.c
--- bonfire-0.3.0.orig/src/burn-dialog.c	2006-05-11 18:11:57.000000000 +0200
+++ bonfire-0.3.0/src/burn-dialog.c	2006-05-16 22:39:38.756780200 +0200
@@ -45,7 +45,7 @@
 #include <gtk/gtklabel.h>
 
 #ifdef HAVE_LIBNOTIFY
-#include <libnotify/notifynotification.h>
+#include <libnotify/notification.h>
 #endif
 
 #include "utils.h"
